tests/phpunit/tests/functions/forceSslAdmin.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Tests for the force_ssl_admin function.
+ *
+ * @group functions.php
+ *
+ * @covers ::force_ssl_admin
+ */#
```

Comment:
```suggestion
 */
```
Looks like your IDE keeps adding this `#`. We don't use that in Core.